### DISCUSSION TITLE
API parity between adapters

### DISCUSSION THREAD

**Discussion by tannerlinsley** [TanStack/table#3841] [General] [answered]
@Mokshit06 The `createTable` export should be reexported from the core like it is in the `react-table` adapter. I think in this case, it should also be called `useTable` regardless of the hook nature.

https://github.com/TanStack/react-table/blob/2fb46f6da60f473d38b86668d244c8df822a2dd7/packages/solid-table/src/index.tsx#L30

**Comment by Mokshit06** [comment, marked as answer]
In solidjs, the functions that wrap reactivity generally use the naming convention `create__` and not `use__`. Maybe we could rename the `createTable` function in this adapter to something else that follows this naming scheme rather than changing it to `useTable`.

**Reply by tannerlinsley** [reply]
https://github.com/TanStack/react-table/commit/28ef10c63aec499f0909996b8b6b1918abbe7b9f

**Reply by Mokshit06** [reply]
Yep that seems better. Should the react adapter also be named `useTableInstance` to match with this?

**Reply by tannerlinsley** [reply]
Good point

**Reply by tannerlinsley** [reply]
Done
